### PR TITLE
[doc](doc)add some k8s doc

### PR DESCRIPTION
**Pull request by LemonLiTree**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by LemonLiTree** [comment]
run buildall

File: docs/zh-CN/docs/install/k8s-deploy/debug-crash.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+```
+当服务进行下一次重启时候，服务会检测到标识 `debug` 模式启动的 annotation 就会进入 `debug` 模式启动，pod 状态为 `running`。
+
+**2.当服务进入 `debug` 模式，此时服务的 pod 显示为正常状态，用户可以通过如下命令进入 pod 内部**
+
+```shell
+$ kubectl --namespace ${namespace} exec -ti ${pod_name} bash
+```
+
+**3. `debug` 下手动启动服务，当用户进入 pod 内部，通过修改对应配置文件有关 HTTP 的端口进行手动执行 `start_xx.sh` 脚本，脚本目录为 `/opt/apache-doris/xx/bin` 下。**
```
Comment (intelligentfu) [inline docs/zh-CN/docs/install/k8s-deploy/debug-crash.md:53]: 这个地方我改了，fe要修改query_port be需要修改heartbeat_service_port。主要是避免debug模式下还能通过service访问到crash的节点导致误导流。

**Comment by LemonLiTree** [comment]
run buildall

File: docs/zh-CN/docs/install/k8s-deploy/debug-crash.md (hunk 1 above)
Comment (LemonLiTree) [reply docs/zh-CN/docs/install/k8s-deploy/debug-crash.md]: done

**Review by intelligentfu — APPROVED** [review]

**Review by CalvinKirs — APPROVED** [review]

**Pull request merged**